packages/react-magma-dom/package.json

```diff
@@ -50,6 +50,7 @@
   "devDependencies": {
     "@emotion/core": "^10.1.1",
     "@emotion/styled": "^10.0.27",
+    "@testing-library/user-event": "^13.2.0",
```

Comment:
testing-library is at the root level... and if they're here, versions must match

Reply:
this is why package-lock showed up